package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@moodlehq/cordova-plugin-qrscanner",
-  "version": "3.0.1-moodle.2",
+  "version": "3.0.1-moodle.3",
```

Comment:
Instead of bumping the version number here, we should do it on a separate release commit. Other than that, I think this is a good candidate to contribute to the original plugin at https://github.com/bitpay/cordova-plugin-qrscanner, so you could open a PR there and then add it to this repo so that we don't have to wait for them to merge it. This is the same we did with https://github.com/bitpay/cordova-plugin-qrscanner/pull/266 as documented in the README.

Reply:
https://github.com/bitpay/cordova-plugin-qrscanner/pull/382